scipy/__init__.py

```diff
@@ -138,8 +138,8 @@ def __dir__():
     from scipy._lib import _pep440
     # In maintenance branch, change to np_maxversion N+3 if numpy is at N
     # See setup.py for more details
-    np_minversion = '1.16.5'
-    np_maxversion = '9.9.99'
+    np_minversion = '1.17.3'
```

Comment:
shouldn't this be forward-ported too?

Reply:
Yes, I will take that along in the PR I'm about to make.

Reply:
done in gh-15197